src/display/process/allProcs.go

```diff
@@ -268,12 +279,44 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 							killSelected = false
 							myPage.ProcTable.CursorColor = selectedStyle
 						}
+						sendSignal = false
+						updateUI()
 					case "K", "<F9>":
 						// get process and kill it
 						procToKill, err := proc.NewProcess(pidToKill)
 						myPage.ProcTable.CursorColor = selectedStyle
 						if err == nil {
-							err = procToKill.Kill()
+							err = procToKill.SendSignal(syscall.SIGTERM)
+							if err != nil {
+								myPage.ProcTable.CursorColor = errorStyle
+							}
+						} else {
+							myPage.ProcTable.CursorColor = errorStyle
+						}
+						runAllProc = true
+						killSelected = false
+						updateProcs()
+					case "j", "<Down>":
+						signals.Table.ScrollDown()
+						ui.Render(signals)
+					case "k", "<Up>":
+						signals.Table.ScrollUp()
+						ui.Render(signals)
+					case "1", "2", "3", "4", "5", "6", "7", "8", "9":
+						scrollIdx, _ := strconv.Atoi(e.ID)
+						if utils.Contains([]string{"1", "2", "3"}, previousKey) != -1 {
+							prevIdx, _ := strconv.Atoi(previousKey)
+							scrollIdx = 10*prevIdx + scrollIdx
+							previousKey = "g" // TODO: don't use this hack
+						}
+						signals.Table.ScrollToIndex(scrollIdx - 1) // account for 0-indexing
+						ui.Render(signals)
```

Comment:
Maybe add a few comments explaining that as number keys are typed, cursor scrolls to that index?

Reply:
I will. Need to work on documenting few of the other places as well